recipes/dpath/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "dpath" %}
+{% set version = "1.4.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "89fbec229d941e622ba4e9578a79e05f8e0b9efdfe40ffbf4e670a129ec54d6d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - dpath
+
+about:
+  home: https://www.github.com/akesterson/dpath-python
+  license: MIT
+  license_family: MIT
+  license_file: ''
+  summary: Filesystem-like pathing and searching for dictionaries
+  description: "dpath-python\n============\n\n|PyPI|\n|Build Status|\n\nA python library for accessing and searching dictionaries via\n/slashed/paths ala xpath\n\nBasically it lets you glob over a dictionary\
```

Comment:
This is gross, and we definitely don't need broken links to the PyPI page or install instructions here. Please change it to something like:
```yaml
  description: |
    A python library for accessing and searching dictionaries via /slashed/paths
    ala xpath. Basically it lets you glob over a dictionary as if it were a filesystem.
    It allows you to specify globs (ala the bash eglob syntax, through some advanced 
    fnmatch.fnmatch magic) to access dictionary elements, and provides some
    facility for filtering those results.
```